plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-coupons-v2-controller.php

```diff
@@ -347,6 +348,11 @@ public function get_item_schema() {
 					'type'        => 'string',
 					'context'     => array( 'view', 'edit' ),
 				),
+				'status'                      => array(
+					'description' => __( 'The status of the coupon. Should always be draft, published or pending review', 'woocommerce' ),
```

Comment:
Minor: Oxford comma https://make.wordpress.org/docs/style-guide/punctuation/commas/#oxford-serial-commas
```suggestion
					'description' => __( 'The status of the coupon. Should always be draft, published, or pending review', 'woocommerce' ),
```